### ISSUE TITLE
Can't install multiples instances "Port '2368' is in use"

### ISSUE THREAD

**Issue by treyssatvincent** [TryGhost/Ghost-CLI#496]
### This issue is a

- [x] Bug Report
* [ ] Feature Request

### Summary

If this is a bug, double-check and make sure there's not already an existing issue for it, then provide a quick summary of what error is occurring.

### Steps to Reproduce (for a bug report)

1. run ``ghost install`` on /var/www/exampleone
2. run ``ghost install` on /var/www/exampletwo

> ? Enter your Ghost database name: ghost_production_fixmystreet
> ✖ Configuring Ghost
> A ConfigError occurred.
> 
> Error detected in the undefined configuration.
> 
> Message: Port '2368' is in use.
> Configuration Key(s): server.port
> Current Value(s): 2368
> 
> Help: Run `ghost config <key> <new value>` for each key to fix the issue.

Debug Information:
    Node Version: 
    Ghost-CLI Version: 1.1.3
    Environment: production
    Command: 'ghost install'

### Technical details (will be automatically output by Ghost-CLI if an error occurs):

* OS: Ubuntu 16.04
* Node Version: v6.11.4
* Ghost-CLI Version: 1.1.3
* Environment: production
* Command: ghost install

**Comment by ErisDS** [comment]
I can confirm this bug, I also ran into it today.

**Comment by TomChantler** [comment]
This is odd as we had an issue a while back with port numbers limiting multiple installations and which was described in #451 and fixed in #452. I wonder why it's broken again? I'll try to take a look.

**Comment by lmuzquiz** [comment]
I also ran into this issue today. 

This is what i did: 

mkdir /var/www/lmuzquiz/blog
cd /var/www/lmuzquiz/blog
ghost install

Then after installing ghost successfully, i  intentionally removed the whole blog folder by running: 

rm -rf /var/www/lmuzquiz/blog

Created the blog folder again, ran ghost install, and the installation process begins smoothly but after i answer this question: 

Enter your Ghost database name: my_ghost_db

That is when i get the error: 

`A ConfigError occurred.

Error detected in the undefined configuration.

Message: Port '2368' is in use.
Configuration Key(s): server.port
Current Value(s): 2368

Help: Run `ghost config <key> <new value>` for each key to fix the issue.


Debug Information:
    Node Version: v6.11.4
    Ghost-CLI Version: 1.1.3
    Environment: production
    Command: 'ghost install'
`

**Comment by TomChantler** [comment]
The way I resolved it was to run `ghost ls` and see which ports were in use (only 2368 in my case). Then I simply ran `ghost install --port 2369` and it worked fine.

**Comment by lmuzquiz** [comment]
If i run ghost ls i get: 
```

──────┬────────────────────────┬─────────┬─────────┬─────┬──────┬─────────────────┐
│ Name │ Location               │ Version │ Status  │ URL │ Port │ Process Manager │
├──────┼────────────────────────┼─────────┼─────────┼─────┼──────┼─────────────────┤
│      │ /var/www/lmuzquiz/blog │ 1.16.1  │ stopped │ n/a │ n/a  │ n/a             │
└──────┴────────────────────────┴─────────┴─────────┴─────┴──────┴─────────────────┘
```

but if run sudo netstat -plotn i get: 

`tcp        0      0 127.0.0.1:2368          0.0.0.0:*               LISTEN      7439/node        off (0.00/0/0)
`

I think that means node is using the port?

Anyway i wil try installing ghost by specifying the port number

**Comment by treyssatvincent** [comment]
@TomChantler That's what I've done, but it was a better behaviour when the CLI changed the installation port itself. This is why I've filled an issue.

**Comment by kirrg001** [comment]
https://github.com/TryGhost/Ghost-CLI/issues/496#issuecomment-339012423

This sounds like a different report and i can reproduce this when i remove my config file in the root folder. Maybe you have removed the config by accident file before running `uninstall`? @ErisDS 

---

Running multiple instances with different ports should be fixed via #526.

**Comment by kirrg001** [comment]
This should be fixed via https://github.com/TryGhost/Ghost-CLI/pull/526. Closing. @ErisDS feel free to open a new issue for https://github.com/TryGhost/Ghost-CLI/issues/496#issuecomment-343546935.

**Comment by dhuang612** [comment]
Hi,

I think this issue is happening again. 
I followed the dev guide setup and ran npm i

after trying to run npm start I get the following error:
```
"Port 2368 is already in use by another program."
"Is another Ghost instance already running?"
```
then running ghost ls
```
ghost ls
No installed ghost instances found
```
then running the above suggested:
```
ghost i --port 2368
returns
A SystemError occurred.
```

**Comment by amithachapa** [comment]
> This should be fixed via #526. Closing. @ErisDS feel free to open a new issue for [#496 (comment)](https://github.com/TryGhost/Ghost-CLI/issues/496#issuecomment-343546935).

How can this be an issue. Changing port while the server has another port open? How stupid is that answer!
